wiki/Documentation-single-source-of-truth.md

```diff
@@ -530,61 +561,15 @@ Modelled process
             "name": "#+# First activity",
             "id": "Activity_0a128t6",
             "predecessorIds": ["StartEvent_1"],
-            "successorIds": ["Activity_0ascvdr"],
-            "outputVariable": ""
+            "successorIds": ["Activity_0ascvdr"]
         },
-        // RPA activity
+        // second activity
         {
             "type": "INSTRUCTION",
             "name": "#+#+# RPA Activity",
             "id": "Activity_0ascvdr",
             "predecessorIds": ["Activity_0a128t6"],
-            "successorIds": ["Event_0wghmrz"],
-            "rpaApplication": "Browser",
-            "rpaTask": "Open Browser",
-            "parameters": [
-                {
-                    "name": "url",
-                    "value": "TestString",
-                    "requireUserInput": true
-                },
-                {
-                    "name": "browser",
-                    "value": "TestString",
-                    "requireUserInput": true
-                },
-                {
-                    "name": "remote_url",
-                    "value": "true",
-                    "requireUserInput": true
-                },
-                {
-                    "name": "desired_capabilities",
-                    "value": "TestString",
-                    "requireUserInput": true
-                },
-                {
-                    "name": "ff_profile_dir",
-                    "value": "TestString",
-                    "requireUserInput": true
-                },
-                {
-                    "name": "options",
-                    "value": "TestString",
-                    "requireUserInput": true
-                },
-                {
-                    "name": "service_log_path",
-                    "value": "TestString",
-                    "requireUserInput": true
-                },
-                {
-                    "name": "executable_path",
-                    "value": "TestString",
-                    "requireUserInput": true
-                },
-            ],
-            "outputVariable": ""
```

Comment:
Why was the RPA activity adapted to be a normal activity? If it is necessary, why wasn't the xml adapted as well? This makes the documentation inconsistent